README-translated/README-Hindi.md

```diff
@@ -0,0 +1,127 @@
+<a href="https://the-turing-way.netlify.app/welcome.html"><img src="../book/website/figures/logo/logo.jpg?raw=true)" width="180" align="Right" /></a>
+
```

Comment:
First review :  Just making the suggestions as they come for some part of the document. If you have time you can already start working- or else- i plan to : 
1. go through the entire document first for initial edits/suggestions- this may already take a few reviews.
2. then later come back to the review (or the already accepted change) and see if i can (if still needed) paste the devnagri script 
3. recheck for finer stylistic edits like use of hyphen, whether english words are italicised or not etc.

feel free to start working on the edits at any stage (or do it at the end, as you prefer)- i'll catch up accordingly